tests/reporters/test_rng_reporter.py

```diff
@@ -145,3 +145,63 @@ async def test_invalid_timestamp_in_future(rng_reporter, monkeypatch, caplog):
         assert status.ok
         assert receipt["status"] == 1
         assert "less than current time" in caplog.text
+
+
+def generate_numbers():
+    current_number = 1678311000
+    while current_number > 1678310000:
+        yield current_number
+        current_number = int(time.time())
+
+
+@pytest.mark.asyncio
+async def test_repeat_auto_rng_report_value(rng_reporter: RNGReporter, monkeypatch, caplog):
+    """Test that shows the bad behavior before bug fix that was causing same value
+    to be reported for timestamps that didn't have block hashes generated yet.
+    Reason is global scope of local_source variable in assemble_rng_datafeed function
+    """
+    # function before fix was applied used local_source var from global scope
+    from telliot_feeds.feeds.tellor_rng_feed import local_source
+
+    async def assemble_feed(timestamp):
```

Comment:
Could import this function from `src/telliot_feeds/feeds/tellor_rng_feed.py` as well?

Reply:
not the same function